build-deps-info/RELEASE.md

```diff
@@ -1,3 +1,3 @@
 ## 1.0.0 (September 11, 2020)
```

Comment:
If we already released 1.0.0, let's release 1.1.0 and have the release noted include both versions.